pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -71,12 +71,17 @@ stdenv.mkDerivation rec {
       sha256 = "0g6fhqcv8spfy3mfmxpyji93k8d4p4q4fz1v9a1c1cgcwkz41d7p";
     })
 
-    # https://gitlab.gnome.org/GNOME/gtk/merge_requests/1002
-    ./patches/01-build-Fix-path-handling-in-pkgconfig.patch
+    # Fix path handling in pkg-config
+    # https://gitlab.gnome.org/GNOME/gtk/merge_requests/1793
```

Comment:
it's soo confusing that Matthias would reject it and then accept this patch later on :grin:

Reply:
You have to present these things as a clean-up, not a compat patch. Also this one is a no-op for normal Meson, your patch made the paths always absolute.

Reply:
:rofl: Sneaky sneaky, make it so they don't have to think about it.